core/src/main/java/org/dst/core/operatorImpl/DstTableImpl.java

```diff
@@ -1,18 +1,228 @@
 package org.dst.core.operatorImpl;
 
-import org.dst.common.exception.NotImplementException;
-import org.dst.core.TableSpecification;
-import org.dst.core.RecordEntry;
 import org.dst.core.operatorset.DstTable;
+import org.dst.core.table.TableEntry;
+import org.dst.core.table.TableSpecification;
+import org.dst.core.table.Record;
+import org.dst.core.table.Value;
+import org.dst.core.table.Field;
+import org.dst.core.table.Index;
+import org.dst.core.table.ValueType;
+import org.dst.common.exception.IncorrectRecordFormatException;
+import org.dst.common.exception.IncorrectTableFormatException;
+import org.dst.common.exception.TableAlreadyExistsException;
+import org.dst.common.exception.TableNotFoundException;
+import java.util.Map;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Arrays;
+import java.util.ArrayList;
 
 public class DstTableImpl implements DstTable {
+
+  private HashMap<String, TableEntry> tableMap;
+
+  public DstTableImpl() {
+    this.tableMap = new HashMap<String, TableEntry>();
+  }
+
   @Override
   public void createTable(TableSpecification tableSpec) {
-    throw new NotImplementException();
+    checkFormatefTableSpecification(tableSpec);
+    if (isExist(tableSpec.getName())) {
+      throw new TableAlreadyExistsException(tableSpec.getName());
+    }
+    TableEntry table = new TableEntry.Builder().tableSpec(tableSpec).builder();
+    tableMap.put(tableSpec.getName(), table);
+  }
+
+  @Override
+  public void append(String tableName, List<Record> sourceRecords) {
+    ArrayList<Record> records = new ArrayList<>(sourceRecords);
```

Comment:
Why is this `ArrayList`? 

I think it's better to use `List`.

Reply:
The purpose of this is to convert to ArrayList(), for the reason that the pass by object may be  not a ArrayList(),   However,the declarations use List is better .i'll fix it